plugins/module_utils/network/ios/utils/utils.py

```diff
@@ -279,6 +280,29 @@ def netmask_to_cidr(netmask):
     return str(sum([bin(int(x)).count("1") for x in netmask.split(".")]))
 
 
+def is_valid_ipv4(ip_str):
+    return (
+        True if re.match(r"(?:[0-9]{1,3}\.){3}[0-9]{1,3}", ip_str) else False
+    )
+
+
+def is_valid_ipv6(ip_str):
```

Comment:
This can probably be replaced with the following:
```s
from ansible.module_utils.compat import ipaddress

valid = True
try:
   ipaddress.ip_address("2001:db8::")
except ValueError:
    valid = False

return valid
```

Reply:
thanks, updated with review suggestion